qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserAccountPage.java

```diff
@@ -74,9 +74,8 @@ public void clickOnAddUserAccount() {
 		clickOn(ADD_USER_ACCOUNT);
 	}
 
-	public void enterUsername(String username) {
-		findElement(USER_NAME).clear();
-		findElement(USER_NAME).sendKeys(username);
```

Comment:
How is this change related to what you are fixing?

Reply:
The method enterUsename() whenever invoked has been using a fixed string so have modified it to accept a random string.

Reply:
By the way retaining the method the way it was also does the work.